web3/utils/transactions.py

```diff
@@ -30,6 +30,18 @@
 }
 
 
+@curry
+def fill_nonce(web3, transaction):
+    if 'from' in transaction and 'nonce' not in transaction:
+        return dict(
```

Comment:
can you change this to use `cytoolz.assoc`

Reply:
*`web3.utils.toolz.assoc`